source/user-manual/agents/query-configuration.rst

```diff
@@ -8,7 +8,7 @@
 Query configuration
 ===================
 
-The actual configuration of an agent, or the manager can be queried on demand by clicking on the **Agents** tab or the **Management** tab. From here you will be able to fetch the active configuration in real time.
+The actual configuration of an agent, or the manager can be queried on demand by clicking on the **IT Hygiene** module or the **Settings** module. From here you will be able to fetch the active configuration in real time.
```

Comment:
Do we mean `Server management > Settings`?

Reply:
I will add this.